packages/aml-checklist/src/components/ui/menu.tsx

```diff
@@ -38,21 +41,21 @@ export const generateMenuConfig = (logoutCallback: () => void, location: Locatio
         url: Routes.RESULTS,
         type: 'PRIMARY',
       },
-      {
+      !isDemo && {
         title: 'Apps',
         key: 'APPS',
         icon: <AppsIcon />,
         callback: () => (window.location.href = window.reapit.config.marketplaceUrl),
         type: 'PRIMARY',
       },
-      {
+      !isDemo && {
         title: 'Logout',
         key: 'LOGOUT',
         callback: logoutCallback,
         icon: <ProfileIcon />,
         type: 'SECONDARY',
       },
-    ],
+    ].filter(Boolean) as MenuItem[],
```

Comment:
Can do a clever trick to do away with the cast. You'll probably decide against it but it's a way around the cast :) 

```ts
const isMenuItem = (menu: any): menu is MenuItem => typeof menu === 'object' && menu.hasOwnProperty('title')

return [menuItem, false].filter<MenuItem>(item => isMenuItem(item))
```

Reply:
But you said the menu was of type any!